src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $time-hover-nav-card;
```

Comment:
Could you use separate `transition-property` and `transition-duration` rules here?